eb_sqs/worker/service.py

```diff
@@ -116,7 +118,9 @@ def process_message(self, msg, worker):
                     receive_count, msg.message_id, msg.body
                 ))
 
-            worker.execute(msg.body)
+            with django_db_management():
```

Comment:
maybe better to do it on `self.process_messages` call?
might be wasteful to do it for every message...

Reply:
I think it's important to do it for each message because we don't know how long each takes to be processed. Django itself doesn't do anything to the connection if it's not broken or outdated so performance impact should be minimal.

Reply:
ok, sounds good

probably worth testing this solves the issue (e.g. by running a worker and rebooting some database)